translations/remote-schemas.kannada.md

```diff
@@ -0,0 +1,83 @@
+# ರಿಮೋಟ್ ಸ್ಕೀಮಾಗಳು
+
+ನೀವು ಎಲ್ಲಾ ಪ್ರಕಾರದ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಅನ್ನು ಒಂದೇ ಸ್ಥಳದಿಂದ ಪ್ರಶ್ನಿಸಲು ಬಯಸಿದರೆ ನಂತರ ದೂರಸ್ಥ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸ್ಕೀಮಾವನ್ನು ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಎಂಜಿನ್‌ನ ಪೋಸ್ಟ್‌ಗ್ರೆಸ್ ಆಧಾರಿತ ಸ್ಕೀಮಾದೊಂದಿಗೆ ವಿಲೀನಗೊಳಿಸಿ. ರಿಮೋಟ್ ಸ್ಕೀಮಾಗಳು ಕೆಲವು ಉದಾಹರಣೆಗಳಿಗೆ ಸೂಕ್ತವಾಗಿವೆ:
+
+- ರೂಪಾಂತರಗಳನ್ನು ಉತ್ತಮಗೊಳಿಸುವುದು (ಉದಾಹರಣೆಗೆ ಒಳಪಡಿಸುವ ಮೊದಲು ಮೌಲ್ಯೀಕರಣ)
+- ಪಾವತಿಗಳು, ಇತ್ಯಾದಿ ವೈಶಿಷ್ಟ್ಯಗಳನ್ನು ಬೆಂಬಲಿಸುವುದು ಮತ್ತು ಅವುಗಳನ್ನು ಪ್ರವೇಶಿಸಲು ಸ್ಥಿರವಾದ ಇಂಟರ್ಫೇಸ್ ಅನ್ನು ಒದಗಿಸುವುದು ಅಂದರೆ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಎಂಜಿನ್‌ನ ಹಿಂದಿನ ಎಪಿಐ
+- ಇತರ ಮೂಲಗಳಿಂದ ಡೇಟಾವನ್ನು ಬೇರ್ಪಡಿಸುವುದು (_ಉದಾಹರಣೆಗೆ ಹವಾಮಾನ ಎಪಿಐ ಅಥವಾ ಇನ್ನೊಂದು ಡೇಟಾಬೇಸ್‌ನಿಂದ_)
+
+ಕಸ್ಟಮ್ ವ್ಯವಹಾರ ತರ್ಕವನ್ನು ಬೆಂಬಲಿಸಲು, ನೀವು ಕಸ್ಟಮ್ ಸರ್ವರ್ ಅನ್ನು ರಚಿಸಬೇಕಾಗುತ್ತದೆ (ನೋಡಿ [ಬಾಯ್ಲರ್‌ಪ್ಲೇಟ್‌ಗಳು](../ಸಮುದಾಯ/ಬಾಯ್ಲರ್‌ಪ್ಲೇಟ್‌ಗಳು/ರಿಮೋಟ್-ಸ್ಕೀಮಾಸ್)) ಮತ್ತು ಅದರ ಸ್ಕೀಮಾವನ್ನು ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಎಂಜಿನ್‌ನೊಂದಿಗೆ ವಿಲೀನಗೊಳಿಸಿ.
```

Comment:
ನೀವು ಕಸ್ಟಮ್ ಸರ್ವರ್ ಅನ್ನು ರಚಿಸಬೇಕಾಗುತ್ತದೆ needs to be changed to ನೀವು ಕಸ್ಟಮ್ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸರ್ವರ್ ಅನ್ನು ರಚಿಸಬೇಕಾಗುತ್ತದೆ